spoken/settings.py

```diff
@@ -12,6 +12,8 @@
 from os.path import *
 from .config import *
 import os
+import datetime
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
```

Comment:
Please BASE_DIR as it is already defined in line no. 19.

Reply:
I think you mean:
"Please **Delete** BASE_DIR as it is already defined in line no. 19."

Reply:
yes